### DISCUSSION TITLE
Event Sourcing .Net Core framework

### DISCUSSION THREAD

**Discussion by tnsivakumar** [oskardudycz/EventSourcing.NetCore#225] [Q&A] [answered]
@oskardudycz, we are looking for a framework (DDD, EventSourcing, CQRS). we have explore another framework but it only supports database as an Event Store not EventStore DB. This code is nice, does it available as a Nuget package or Do you have any recommendation? Thanks in advance

**Comment by oskardudycz** [comment, marked as answer]
@tnsivakumar I’m glad that you like my samples. 👍

Regarding publishing core projects as NuGet packages, some older versions are published as GoldenEye https://github.com/oskardudycz/GoldenEyeyet, I stopped (for now) aligning it as I cannot dogfood it currently and don’t have a financial incentive to do it for free. I’d consider that if I got sponsorship for that.

If you’re searching for a framework, I’d suggest have a look at Wolverine ing a look at Wolverine (https://wolverine.netlify.app/) and Eventuous (https://eventuous.dev). They’re both well-thought and maintained actively.

Of course, you can use also this samples as-is as a base for your own core.
